app/Console/Commands/Setup/SetupMailCommand.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace App\Console\Commands\Setup;
+
+use App\Classes\Pterodactyl\PterodactylClient;
+use App\Settings\MailSettings;
+use GuzzleHttp\Promise\PromiseInterface;
+use Illuminate\Console\Command;
+use Illuminate\Http\Client\Response;
+use Illuminate\Support\Facades\Hash;
+use Illuminate\Support\Facades\Validator;
+use Spatie\Permission\Models\Role;
+use function __;
+
+class SetupMailCommand extends Command
+{
+    protected PterodactylClient $client;
+
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 's:mail:create';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Create the email settings from the .env file';
+    private MailSettings $settings;
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct(PterodactylClient $client, MailSettings $settings)
+    {
+        parent::__construct();
+        $this->client = $client;
+        $this->settings = $settings;
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle()
```

Comment:
i personally think having a command to set-up email details is not nesseary since you can do this once you are logged in in the back-end which is guided by a UI and validations. plus test button. which is i think more user friendly so a command would be pretty double.

Reply:
So no set-up command, but then how can you login if the original user gets a welcome message? Should I remove the whole welcome email function?